scripts/update/00-run.sh

```diff
@@ -39,8 +39,9 @@ RESOLV_CONF_BACKUP_FILE="/tmp/resolv.conf"
 cat "${RESOLV_CONF_FILE}" > "${RESOLV_CONF_BACKUP_FILE}"
 
 # Use Cloudflare and Google public DNS servers for update
-echo "nameserver 1.1.1.1" > "${RESOLV_CONF_FILE}"
+echo "nameserver 1.1.1.1" >> "${RESOLV_CONF_FILE}"
 echo "nameserver 8.8.8.8" >> "${RESOLV_CONF_FILE}"
+echo "options timeout:1" >> "${RESOLV_CONF_FILE}"
```

Comment:
I like the general idea, but do you know the behaviour where they already have other DNS config? For example, say they have currently:

```
nameserver {umbrel_server_ip_address}
options timeout:2 rotate attempts:3
```

And then we append Public DNS servers as a backup, to become:
```
nameserver {umbrel_server_ip_address}
options timeout:2 rotate attempts:3
nameserver 1.1.1.1
nameserver 8.8.8.8
options timeout:1
```

Will the `options` simply override previous settings?